packages/trader/src/sass/app/_common/layout/trader-layouts.scss

```diff
@@ -432,6 +414,14 @@ $FLOATING_HEADER_HEIGHT: 41px;
                 }
             }
         }
+        &--screenshot {
+            .sc-toolbar-widget,
+            .stx-panel-control,
+            .ciq-navigation-widget,
+            .cq-top-ui-widgets {
+                transform: translate3d(0, 0, 0) !important;
```

Comment:
is this added to leverage gpu rendering?

Reply:
@reza-fs no, actually if user opens the left panel, these items move to the right side, and on screenshot, we need them to get back to the left side. so only when we do the screenshot we ad `ciq-chart--screenshot` and then remove it.